### ISSUE TITLE
IBDesignable not working properly

### ISSUE THREAD

**Issue by hlarcher** [hsousa/HCSStarRatingView#65]
...storyboard: error: IB Designables: Failed to render and update auto layout status for ViewController (CvE-oC-4JB): Failed to load designables from path (null)

installing via pod this happens every time I try to use this class in interface builder and then everything disappears (doesnt render) from the view.

**Comment by yurii-petrov** [comment]
<img width="235" alt="screen shot 2018-04-09 at 00 41 03" src="https://user-images.githubusercontent.com/20144003/38472852-bf2526e6-3b8e-11e8-94ea-eeb6e16a8494.png">
I've got the same problem.

**Comment by hsousa** [comment]
Did you add `use_frameworks!` do your podfile?

**Comment by Aryal-Nischal** [comment]
Yes `use_frameworks!` is enabled in podfile. Still receiving the issue mentioned by @petrovRV

**Comment by bkvocka** [comment]
Downgrading cocoapods to 1.4.0 fixed this problem.

**Comment by paulja** [comment]
Any further news on this? The error I receive is: 

```
file:///.../blah.storyboard: error: IB Designables: Failed to render and update auto layout 
status for ItemController (zT7-9g-pf6): dlopen(HCSStarRatingView.framework, 
1): no suitable image found.  Did find:
	HCSStarRatingView.framework: required code signature 
        missing for 'HCSStarRatingView.framework'
```

Seems to be some kind of permissions/code signing issue?

**Comment by ashishpisey** [comment]
Any updates on this ?
